steampy/models.py

```diff
@@ -36,6 +36,7 @@ class Currency(enum.IntEnum):
     GBP = 2
     EURO = 3
     CHF = 4
+    RUB = 5
```

Comment:
Please remove this as we have know currncies in models and it is creating merge conflict